src/styles/main.scss

```diff
@@ -1,3 +1,50 @@
-body {
-  margin: 0;
+@import "./variables.scss";
+@import "./reset";
+@import "./normalise";
+
+.month {
+  width: 7*$cell-size + $cell-size/10*2 + 6px;
```

Comment:
You can transfer mathematical operations to separate variables and use the result immediately, this will increase readability.